inscriptions/ISic001438.xml

```diff
@@ -229,9 +233,7 @@
                         <citedRange>150 no.15 fig.33.2</citedRange>
                         <ptr target="https://www.zotero.org/groups/382445/items/UUNP76BF"/>
                     </bibl>
-                    <bibl>
-                        <author>Arena</author>
-                        <date>1996</date>
+                    <bibl type="corpus" n="IGASM1">
```

Comment:
For the moment I am not changing the abbreviation. About what you say:
(a) we initially decided to keep the abbreviations as you set them up, rather than conform to the new ones from the AIEGL list. I defer to you on this;
(b) I would still include abbreviation + author + date so as not to make confusion, even if it is the reference in Zotero that counts.

Reply:
I'm glad you remember what I said... Excellent, let's do that then.